### DISCUSSION TITLE
How to prevent scroll lock on Dropdown and similar components?

### DISCUSSION THREAD

**Discussion by ojvribeiro** [unovue/reka-ui#542] [Q&A] [answered]
I see that we have the [Config Provider](https://www.radix-vue.com/utilities/config-provider.html), but the `scrollBody` prop set to false doesn't seem to work.

**Comment by adirizky54** [comment, marked as answer]
I think you can set prop `modal` to false on DropdownMenuRoot

**Reply by zernonia** [reply]
The `scrollBody` only affect the size of scrollbar. You should use the `modal` as @adirizky54  suggested 😁

**Reply by ojvribeiro** [reply]
Oh, that worked! Thanks a lot. ♥
